contents/dataframes_indexing.md

```diff
@@ -69,35 +70,37 @@ collect(zip(df.name, df.grade_2020))
 """)
 ```
 
-However, converting a DataFrame to a Dictionary is only useful when the elements are unique.
-When that is not the case, it is time to `filter` (@sec:filter).
+However, converting a `DataFrame` to a `Dict` is only useful when the elements are unique.
+Generally that is not the case, it is time to learn how to `filter` a `DataFrame`.
 
 ## Filter and Subset {#sec:filter_subset}
 
-There are two ways to remove rows from a DataFrame, one is `filter` (@sec:filter) and the other is `subset` (@sec:subset).
+There are two ways to remove rows from a `DataFrame`, one is `filter` (@sec:filter) and the other is `subset` (@sec:subset).
 `filter` was added earlier to `DataFrames.jl`, is more powerful, and more consistent with syntax from Julia base, so that is why we start with discussing `filter`.
 `subset` is newer and often more convenient.
 
 ### Filter {#sec:filter}
 
-From this point on, we start to get to the more eowerful features of `DataFrames.jl`.
-To do this, we need to learn some verbs, such as `select` and `filter`, but it might be a relieve to know that the general design goal of `DataFrames.jl` is to keep the number of verbs that a user has to learn to a minimum[^verbs].
-Continuing from the earlier mentioned data:
+From this point on, we start to get into the more powerful features of `DataFrames.jl`.
+To do this, we need to learn some functions, such as `select` and `filter`.
```

Comment:
I copied the word "verbs" from bkamins: https://discourse.julialang.org/t/pull-dataframes-columns-to-the-front/60327/5.

I agree though that the word "functions" is clearer, but could you doublecheck by looking at the discourse post?

Reply:
Added complexity... Verbs and Functions here can be exchangeable.

I don't think we would benefit to strictly use the word verbs and then having to define what a verb is to the reader.